app/client/src/components/shared/AboutContent/index.js

```diff
@@ -281,8 +285,10 @@ function AboutContent({ ...props }: Props) {
               <div className="container">
                 <h1>Questions and Answers about How’s My Waterway </h1>
                 <hr />
+
+                <h1>What is “How’s My Waterway?”</h1>
```

Comment:
```suggestion
                <h2>What is “How’s My Waterway?”</h2>
```

I think this should be an `<h2>`, along with all of the `<h1>` tags below, since it is a child of "Questions and Answers about How’s My Waterway". 

I tested using an `<h2>` from the Chrome dev tools and though the `<strong>` tags from the previous version looked better. Could you also style them like the previous `<strong>` tags?

Reply:
Thanks, I agree. Made those changes

Reply:
Thanks for making those changes.